vxfreeswitch/voice.py

```diff
@@ -336,10 +341,17 @@ def register_client(self, client):
 
     def deregister_client(self, client, duration=None):
         client_addr = client.get_address()
+
+        # If originated call has not yet been answered
+        if self._unanswered_channels.get(client_addr):
+            d = self._unanswered_channels[client_addr]
+            d.errback(FreeSwitchClientError('Call is unanswered'))
```

Comment:
Also will we not always send an outbound message? https://github.com/praekelt/vumi-freeswitch-esl/blob/develop/vxfreeswitch/voice.py#L331